viewer/test-utilities/src/createDetermineSectorInput.ts

```diff
@@ -18,9 +18,7 @@ export function createDetermineSectorInput(
     loadingHints: {},
     cameraInMotion: false,
     budget: budget || {
-      geometryDownloadSizeBytes: 20,
       highDetailProximityThreshold: 10,
-      maximumNumberOfDrawCalls: Infinity,
       maximumRenderCost: Infinity
```

Comment:
Should this one be less than infinite?

Reply:
Well, it's just for testing - and behaviour is not changed from previous. Might be that it would have made sense with some other values, but I think its fine as is as I dont want to fix tests that rely on this.